### ISSUE TITLE
t/sync.t currently fails

### ISSUE THREAD

**Issue by eserte** [xsawyerx/www-xkcd#3]
```
Can't fetch http://imgs.xkcd.com/comics/: Not Found at t/sync.t line 38.
# Looks like you planned 18 tests but ran 5.
# Looks like your test exited with 255 just after 5.
t/sync.t ................. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 13/18 subtests 
```

Last success report on cpantesters was before about a week, since yesterday there are just fail reports:
http://fast-matrix.cpantesters.org/?dist=WWW-xkcd;reports=1#sl=7,1

**Comment by ambs** [comment]
2016 are all PASS, so I think this cn be safely closed.

**Comment by xsawyerx** [comment]
That's not the problem @eserte's test shows. His test shows that a request was made to a URL of a comic without the comic ID number. This means there was either a problem with the test or with the code which led to a bad request.

That needs to be researched and fixed.

**Comment by eserte** [comment]
Currently there seems to be a https vs. http issue. With WWW-xkcd-0.008:
```
#   Failed test 'Correct img'
#   at t/fetch_metadata_fail.t line 49.
#          got: 'https://xkcd.com/100/info.0.json'
#     expected: 'http://xkcd.com/100/info.0.json'
# Looks like you planned 8 tests but ran 6.
# Looks like you failed 1 test of 6 run.
t/fetch_metadata_fail.t .. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 3/8 subtests 
```

**Comment by xsawyerx** [comment]
I fixed that and tried to release a new version but I cannot seem to get any response with AnyEvent::HTTP. Might be a certain version that doesn't work. Haven't had the time to figure what the problem is. :/

**Comment by xsawyerx** [comment]
Fixed and released in 0.009. Thank you, @eserte.
